pulp_smash/pulp_smash_cli.py

```diff
@@ -41,7 +41,7 @@ def settings(ctx):
 
 @settings.command('create')
 @click.pass_context
-def settings_create(ctx):
+def settings_create(ctx):  # pylint:disable=too-many-locals
```

Comment:
We are now one too many locals, should we make helper function per major config key (pulp, systems)?

Reply:
I'm willing to temporarily mask `too-many-locals`, so long as an issue is filed with a plan for fixing this. One helper function per major config key is a good idea. The Pulp version (2 or 3) may make a difference as to the layout of `systems` in the future, anyway.